src/nodes/accessors/InstanceNode.js

```diff
@@ -265,8 +247,94 @@ class InstanceNode extends Node {
 
 		}
 
+		if ( this.previousInstanceMatrixNode !== null ) {
+
+			frame.object.previousInstanceMatrix.array.set( this.instanceMatrix.array );
+
+		}
+
 	}
 
+	/**
+	 * Computes the transformed/instanced vertex position of the previous frame.
+	 *
+	 * @param {NodeBuilder} builder - The current node builder.
+	 * @return {Node<vec3>} The instanced position from the previous frame.
+	 */
+	getPreviousInstancedPosition( builder ) {
+
+		const instancedMesh = builder.object;
+
+		if ( this.previousInstanceMatrixNode === null ) {
+
+			instancedMesh.previousInstanceMatrix = this.instanceMatrix.clone();
+
+			this.previousInstanceMatrixNode = getInstanceMatrixNode( instancedMesh.previousInstanceMatrix, this.isStorageMatrix );
+
+		}
+
+		return this.previousInstanceMatrixNode.mul( positionPrevious ).xyz;
+
+	}
+
+	/**
+	 * Returns `true` if instance matrices from the previous frame are required. Relevant
+	 * when computing motion vectors with {@link VelocityNode}.
+	 *
+	 * @param {NodeBuilder} builder - The current node builder.
+	 * @return {boolean} Whether instance matrices from the previous frame are required or not.
+	 */
+	needsPreviousInstanceMatrices( builder ) {
```

Comment:
We need the exact same helper in `InstanceNode` and `SkinningNode`. Could we move this to `Node` and rename it to `needsPreviousData()`?